src/utils/craftLayerURL.js

```diff
@@ -10,8 +12,15 @@ export const createLayerURL = (location, type) => {
     pathname,
     search,
   } = location;
+  const pathList = pathname.split('/');
 
-  const url = `${pathname}${search}`;
+  if (type === LAYER_TYPES.CREATE) {
+    pathList.length = 4;
```

Comment:
This should not be necessary. `.push()` automatically increases the array size. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push

Reply:
@ryandberger Tried a new approach, please, review